From 1281b8c94e91d4214272dcff8417fcd6f706ce52 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Thu, 21 Mar 2019 15:35:10 -0700
Subject: [PATCH] MANTA-4172 zk client ping timeouts not working MANATEE-375
 Update dev zone scripts for MANATEE-335 MANATEE-343 manatee full rebuilds
 should pull from the primary where possible MANATEE-400 Manatee deleterious
 when dataset will not unmount MANATEE-405 snapshotter should not enter
 maintenance when dataset does not exist MANATEE-307 manatee-snapshotter
 failing on snapshot with dependent clones MANATEE-332 manatee-snapshotter
 fails spuriously during rebuild MANATEE-397 `manatee-adm history` fails with
 `TypeError: Cannot read property 'zoneId' of null` (again) MANATEE-398
 manatee-adm history -j doesn't understand v1 timestamps MANATEE-389 deposed
 array is missing in stateBackfill MANATEE-390 check for deposed array's
 existence in manatee-adm history MANATEE-359 manatee reports primary to sync
 replication as async MANATEE-341 "manatee-adm rebuild" hangs when rebuilding
 a sync peer MANATEE-344 Rebuild should not exit if postgres hasn't started on
 its first attempt

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 4db2cea..7064eaf 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "carrier": "0.2.0",
-        "manatee": "git+https://github.com/joyent/manatee.git#6a769d97"
+        "manatee": "git+https://github.com/joyent/manatee.git#33b33433070e1c83622ca6a555b4bc38f1eec0e6"
     },
     "sdcDependencies": {
         "config-agent": ">=1.2.0"
-- 
2.21.0

